packages/functional-tests/tests/signin/relyingParties.spec.ts

```diff
@@ -149,49 +149,3 @@ test.describe('severity-3 #smoke', () => {
     await expect(page.locator('text=Sign In/Up').first()).toBeVisible();
   });
 });
-
-test.describe('OAuth and Fx Desktop handshake', () => {
```

Comment:
~Was this test unneeded? Or just flaky? Or should we just add `.skip()` if we plan on keeping it in the future?~

Nevermind, it looks like it might have been migrated to packages/functional-tests/tests/syncV3/signIn.spec.ts below&hellip;

Reply:
Yea this is intended, I found that running Sync related tests together worked slightly better.